polylogue/ui/facade.py

```
@@ -2,10 +2,8 @@
 
 import shutil
 import subprocess
-import sys
 from dataclasses import dataclass, field
-from pathlib import Path
-from typing import Iterable, List, Optional, Protocol, Sequence
+from typing import Iterable, List, Optional, Protocol
```

Ruff (0.15.9)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)